test/test_tesseract.jl

```diff
@@ -52,6 +53,26 @@ function test_run_with_kwargs()
     @test strip(TEST_ITEMS["noisy"]["text"]) != strip(result_txt)
 end
 
+function test_tsv()
+    tmp_path = tempname()
+    mkdir(tmp_path)
+    path_to_res = joinpath(tmp_path, "res.tsv")
+    @test run_tesseract(TEST_ITEMS["simple"]["path"], path_to_res, "tsv"; psm=11 #= sparse =#, lang="eng")
+
+    table, header = readdlm(path_to_res, '\t'; header=true, quotes=false)
+    @test header == ["level"  "page_num"  "block_num"  "par_num"  "line_num"  "word_num"  "left"  "top"  "width"  "height"  "conf"  "text"]
```

Comment:
In order to keep the code style rule of max line length, what if we break the line like this?:

```suggestion
    @test vec(header) == ["level", "page_num", "block_num", "par_num", "line_num",
        "word_num", "left", "top", "width", "height", "conf", "text"]
```